test/README.md

```diff
@@ -0,0 +1,65 @@
+# Performance Tests
+
+# Rendering Tests
+
+Tests for rendering components, which compares image result and stored reference image.
+Tests are run in the browser headles mode, uses `mocha-webdriver-runner` to run `mocha` tests
+in browser context.
+
+## What to do first? Where to start?
+
+Before start the work on implementing new features you should run rendering tests first to
+generate actual images and store them as reference images to have possibility of making comparision
+with right images after doing some changes in code.
+
+### Prerequisites
+-   Linux
+-   Docker installed
+
+### Tests execution - build and run locally
+
+1. Run command:
+
+```shell
+    bash scripts/docker-tests-runner.sh
+    #OR
+    ./scripts/docker-tests-runner.sh
+```
+ 
+2. Local reference images
+
+Now, the problem is that your browser platform string doesn't look like the one that CI uses
```

Comment:
This text doesn't apply as we don't have CI.
You should describe local flow only, where first run just skips all tests because there are no reference images.